test/verify/check-storage-mdraid

```diff
@@ -162,6 +163,11 @@ class TestStorage(StorageCase):
 
         wait_degraded_state(False)
 
+        # HACK - wait a bit before stopping newly created device
+        # https://github.com/storaged-project/udisks/issues/643
+        if m.image == "debian-testing":
+            sleep(10)
```

Comment:
FTR, from reading your upstream bug report, `udevadm settle` might also do the trick. But it's a hack either way, so let's see how this fares.